config/crs/kubernetes/result/operator_v1alpha1_result_cr.yaml

```diff
@@ -18,3 +18,10 @@ metadata:
   name: result
 spec:
   targetNamespace: tekton-pipelines
+  logs_api: true
```

Comment:
We need to do this to avoid needless complexity and to have better UX. Results config is analogous to snake case, not camelCase, that's why we did this.

If you look into other properties, we are already doing this. We are using kebab case, dot case format, etc. It depends upon what properties we want to configure and if they are in what case they are available in projects.

Reply:
fine. What I meant to say is, we have used `targetNamespace` in `camelCase` and other names are in `snake_case`

Reply:
Yes but targetNamespace is coming from every other properties.

Reply:
ok, I just had a quick look and I see for now `-`, `.`, `camelCase` only used. `snake_case` not used in other CRs.
May be we can go with `-` (hyphen). introducing `_`(underscore) may lead confusions between `-` and `_` to user

Reply:
i am not sure if we have spec field like this? you can use json tag for data nin these field. but struct field would be better with camel case